src/components/List.jsx

```diff
@@ -22,10 +48,25 @@ export default function List() {
   return (
     <>
       <h1>Shopping List</h1>
+
       <div>
-        {docs.map((item, index) => {
-          return <p key={index}>{item.data().item}</p>;
-        })}
+        <ul>
+          {docs.map((item, index) => {
+            return (
+              <li key={index}>
+                <input
+                  aria-label="checkbox for purchased item"
+                  id={item.data().id}
+                  type="checkbox"
+                  onChange={(e) => handleCheckBox(e, item)}
+                  checked={check24Hrs(item)}
+                  disabled={check24Hrs(item)}
```

Comment:
Naming is one of the hardest things in software. In this case it's unclear when reading this prop and function name (`checked={check24Hrs(item)}`) what's happening within the function. Translated to a sentence I'd read this as "This input should show as checked if item checked 24 hours", which needs some thought and likely a read of the function body to understand. A more expressive function name with the prop to tell the reader what's happening could be:
```suggestion
                  checked={wasPurchasedWithin24Hours(item)}
                  disabled={wasPurchasedWithin24Hours(item)}
```
This would read to me as "Show this input as checked if it was purchased within 24 hours." More verbose but explains clearly to the reader what the function is doing with the item passed in without reading the function definition

Reply:
Good call out! This is much easier to understand what this function is doing. Just pushed an update for this.